code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -73,6 +73,9 @@
 
```

Comment:
unrelated but i forgot to remove my edits to the gunsgalore dmi so if you could that'd be great